docs/siem/siem-ui.asciidoc

```diff
@@ -1,102 +1,169 @@
+
+Here's an overview of the flow and its components:
+
+* Data is shipped from your hosts to {es} via beat modules and the Elastic
+Endpoint Security agent integration:
+** {security-guide}/install-endpoint.html[{es-sec-endpoint}] - Elastic agent integration that
+protects your hosts {security-guide}/detection-engine-overview.html#malware-prevention[against malware] and ships these data sets:
+***  *Windows*: Process, network, file, DNS, registry, DLL and driver loads,
+malware security detections
+*** *Linux/macOS*: Process, network, file
+** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
+are lightweight data shippers. Beat modules provide a way of collecting and
+parsing specific data sets from common sources, such as cloud and OS events,
+logs, and metrics. Common security-related modules are listed {security-guide}/ingest-data.html#enable-beat-modules[here].
+* The {security-app} in {kib} is used to manage the *Detection engine*,
+*Cases*, and *Timeline*, as well as administer hosts running Endpoint Security:
+** Detection engine: Automatically searches for suspicious host and network
+activity via the following:
+*** {security-guide}/detection-engine-overview.html#detection-engine-overview[Detection rules]: Periodically search the data
+({es} indices) sent from your hosts for suspicious events. When a suspicious
+event is discovered, a detection alert is generated. External systems, such as
+Slack and email, can be used to send notifications when alerts are generated.
+You can create your own rules and make use of our {security-guide}/prebuilt-rules.html[prebuilt ones].
+*** {security-guide}/detections-ui-exceptions.html [Exceptions]: Reduce noise and the number of
+false positives. Exceptions are associated with rules and prevent alerts when
+an exception's conditions are met. *Value lists* contain source event
+values that can be used as part of an exception's conditions. When
+{es-sec-endpoint} is installed on your hosts, you can add malware exceptions
+directly to the endpoint from the Security app.
+*** {security-guide}/machine-learning.html#included-jobs[{ml-cap} jobs]: Automatic anomaly detection of host and
+network events. Anomaly scores are provided per host and can be used with
+detection rules.
+** {security-guide}/timelines-ui.html[Timeline]: Workspace for investigating alerts and events.
+Timelines use queries and filters to drill down into events related to
+a specific incident. Timeline templates are attached to rules and use predefined
+queries when alerts are investigated. Timelines can be saved and shared with
+others, as well as attached to Cases.
+** {security-guide}/cases-overview.html[Cases]: An internal system for opening, tracking, and sharing
+security issues directly in the Security app. Cases can be integrated with
+external ticketing systems.
+** {security-guide}/admin-page-ov.html[Administration]: View and manage hosts running {es-sec-endpoint}.
```

Comment:
Flagging because this is showing up as `{es-sec-endpoint}` instead of `Endpoint Security` in the output.